### ISSUE TITLE
A note about saving long strings

### ISSUE THREAD

**Issue by jmcrey** [justinlee799/pwp-capstones#2]
So, at the beginning of our project, we are attempting to save some very long strings into variables. These strings even contain some special characters, which we must escape to allow for proper processing (nice job escaping them properly by the way). I just wanted to quickly give an alternative to going through and escaping all these characters. 

First, one way to save a very long string is by encapsulating it inside a triple quote, like so:

`murder_note = """This is just a test showing that we don't need to escape characters when saved like this"""`

Notice that we do not have to escape our tick mark with a backslash when using this method. Another way to save a string we know will be having the tick (`'`) mark is simply by using a double quote instead of the tick. For example, this is equally acceptable to the above:

`murder_note = "This is just a test showing that we don't need to escape characters when saved like this"`

Notice that when we use the double quote, we don't need to escape the tick mark. That being said, if our string had a double quote inside it, we would need to escape that using the backslash (`\`) as we have in our original string (of course, in that case, we can simply save it using the tick mark, but I digress). 

That being said, really this is just for easy of saving long strings. Ultimately, any method is perfectly acceptable if you are willing to escape the characters properly (as was done in the original string).

**Comment by justinlee799** [comment]
Yes, I noticed after I should've just used the double "" marks instead. Thank you, will make sure to keep that in mind.
